pkg/api/policies.go

```diff
@@ -0,0 +1,50 @@
+package api
+
+type InferencePropertyPolicies struct {
+	Remote InferenceProviderPolicies `toml:"remote,omitempty"`
+	Public InferenceProviderPolicies `toml:"public,omitempty"`
+}
+
+type InferenceProviderPolicies struct {
+	Enabled *bool `toml:"enabled,omitempty"`
```

Comment:
We might want to add comments to clarify what the values mean.

By our context (and checking the tests) I understand that a nil value means that the policy is enabled (defaults to true), but that isn't that straightforward (especially if you don't see that this is a pointer)